packages/program/src/types.ts

```diff
@@ -12,6 +12,8 @@ export type ContractCall = {
   contractId: AbstractAddress;
   data: BytesLike;
   fnSelector: string;
+  fnSelectorBytes: Uint8Array;
+  encoding?: string;
```

Comment:
Should we make `encoding` a specific type like so instead of keeping it open to a `string`?

```ts
type EncodingScheme = 'v1' | 'v2';

...
encoding?: EncodingScheme;
```

Could also be an enum.

Reply:
https://github.com/FuelLabs/fuels-ts/pull/1832/commits/8c938e8f25fc3d62503c99501d6caf5c2f17f9fd